src/pages/projects/index.page.tsx

```diff
@@ -93,8 +93,8 @@ function Projects() {
     const _cv: CV[] = []
     if (includeV1) _cv.push(CV_V1)
     if (includeV1_1) _cv.push(CV_V1_1)
-    if (includeV2) _cv.push(CV_V2)
-    return _cv.length ? _cv : [CV_V1, CV_V1_1, CV_V2]
+    if (includeV2) _cv.push(CV_V2, CV_V3)
```

Comment:
ohh nice addition. i'm thinking maybe we should change the UI for that version picker to be more explicit about including v3, though that can be another PR

Reply:
yeah totally agree, just went with this for completeness and speed but we should prob add another checkbox 👍